ash/src/extensions/experimental/amd.rs

```diff
@@ -657,11 +657,18 @@ impl<'a> ::std::ops::Deref for PhysicalDeviceWaveLimitPropertiesAmdBuilder<'a> {
     }
 }
 impl<'a> PhysicalDeviceWaveLimitPropertiesAmdBuilder<'a> {
-    pub fn next<T>(mut self, next: &'a mut T) -> PhysicalDeviceWaveLimitPropertiesAmdBuilder<'a>
+    pub fn push_next<T>(
```

Comment:
I'm confused about where this definition is coming from. It's missing the chain support added to the generated version. I guess it's hand-written? Is that necessary?

Reply:
Yes it is handwritten. I am not sure what we should do about it. Maybe we should remove all the  builder stuff because it will probably get out of date anyway?

I am going to update it anyway.

Everything in experimental won't adhere to semver, so we can remove some parts in the future. I probably should document that.

Reply:
Not a blocker, then, but why isn't this generated?

Reply:
Oh this is not included in the vk.xml. That is why it is inside the experimental module. @msiglreith basically reversed the API :). This will be deleted when it is officially included in the vk.xml.